assets/airports/kcvg.json

```diff
@@ -0,0 +1,4741 @@
+            "rwy": {
+                "KCVG9": [],
+                "KCVG18L": [],
+                "KCVG18C": [],
+                "KCVG18R": [],
+                "KCVG27": [],
+                "KCVG36L": [],
+                "KCVG36C": [],
+                "KCVG36R": []
+            },
+            "draw": [
+                ["BVT*", "TORIO", "MACES", "VHP", "SHB", "CEGRM", "CVG"],
+                ["OBK*", "DELHI", "CGT", "CADIZ"],
+                ["JOT*", "CADIZ", "FATOX", "BVT"]
+            ]
+        },
+        "TIGRR2": {
+            "icao": "TIGRR2",
+            "name": "Tiger Two",
+            "entryPoints": {
+                "APE": [],
+                "CTW": ["CTW"]
+            },
+            "body": ["APE", "BOWRR", "ORLEE", "HEDEE", "JARSO", "TIGRR", "CHARZ", "BACAS", "HANUT", "^HAVLO", "#184"],
+            "rwy": {
+                "KCVG9": [],
+                "KCVG18L": [],
+                "KCVG18C": [],
+                "KCVG18R": [],
+                "KCVG27": [],
+                "KCVG36L": [],
+                "KCVG36C": [],
+                "KCVG36R": []
+            },
+            "draw": [
+                ["APE*", "BOWRR", "ORLEE", "HEDEE", "JARSO", "TIGRR", "CHARZ", "BACAS", "HANUT", "HAVLO"],
+                ["CTW*", "APE"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG18L.BNGLE4.DJB",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.2,
+            "airlines": [
+                ["aal", 1],
+                ["aay", 4],
+                ["abx", 10],
+                ["ash", 4],
+                ["cks", 10],
+                ["dal", 15],
+                ["edv", 24],
+                ["eny", 11],
+                ["fdx", 6],
+                ["fft", 10],
+                ["gjs", 5],
+                ["gti", 8],
+                ["jia", 10],
+                ["n/fastGA", 7],
+                ["pac", 5],
+                ["rpa", 8],
+                ["skw", 3],
+                ["swa", 6],
+                ["ual", 3]
+            ]
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG18L.CHCLL4.IIU",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.4,
+            "airlines": [
+                ["aal", 1],
+                ["aay", 4],
+                ["abx", 10],
+                ["ash", 4],
+                ["cks", 10],
+                ["dal", 15],
+                ["edv", 24],
+                ["eny", 11],
+                ["fdx", 6],
+                ["fft", 10],
+                ["gjs", 5],
+                ["gti", 8],
+                ["jia", 10],
+                ["n/fastGA", 7],
```

Comment:
The fastGA are a bit slow in descending from my observations during the practical assessment. Would it be better to add  separate spawnPatterns for the `n/fastGA` airline/fleet, with a spawn altitude of around 12,000ft?